### PR TITLE
Aml 144 142

### PR DESCRIPTION
**Pull request by nielsuit227**

### CODE REVIEW

File: Amplo/AutoML/DataSampler.py [hunk 1]
```diff
@@ -175,25 +177,25 @@ def fit(self, x: pd.DataFrame, y: pd.Series):
         # Get samplers
         samplers = self.get_samplers(y)
 
-        # If samplers are empty, we don't need sampling
+        # If samplers are empty, we don"t need sampling
```
Comment (tstreule) [inline Amplo/AutoML/DataSampler.py:180]: This doesn't look nice

File: Amplo/AutoML/IntervalAnalyser.py [hunk 2]
```diff
@@ -31,7 +31,7 @@ def __init__(self,
         - Remove healthy data in longer, faulty logs
         - Remove redundant data in large datasets
 
-        Uses Facebook's FAISS for K-Nearest Neighbors approximation.
+        Uses Facebook"s FAISS for K-Nearest Neighbors approximation.
```
Comment (tstreule) [inline Amplo/AutoML/IntervalAnalyser.py:34]: Here too

File: Amplo/AutoML/IntervalAnalyser.py [hunk 3]
```diff
@@ -108,7 +108,7 @@ def data_with_noise(self) -> pd.DataFrame:
 
         Notes
         -----
-        Depending on the dtype of the noise attribute, the dtype of the labels' column will be affected.
+        Depending on the dtype of the noise attribute, the dtype of the labels" column will be affected.
```
Comment (tstreule) [inline Amplo/AutoML/IntervalAnalyser.py:111]: Here

File: Amplo/Pipeline.py [hunk 4]
```diff
@@ -51,72 +51,114 @@ def __init__(self, **kwargs):
...
+        interval_analyse : bool
+            Whether to use IntervalAnalyser module
             Note that this has no effect when data from ``self._read_data`` is not multi-indexed
 
         Sequencing:
-        sequence [bool]: [Sequencing] Whether to use Sequence module
-        seq_back [int or list[int]]: Input time indices
+        sequence : bool
+            [Sequencing] Whether to use Sequence module
+        seq_back : int | list[int]
```
Comment (tstreule) [inline Amplo/Pipeline.py:105]: Write instead: int or list of int

File: Amplo/Pipeline.py [hunk 5]
```diff
@@ -51,72 +51,114 @@ def __init__(self, **kwargs):
...
             with exec, can be multiline. Uses data as input.
 
         Flags:
-        logging_level [Optional[Union[int, str]]]: Logging level for warnings, info, etc.
-        plot_eda [bool]: Whether to run Exploratory Data Analysis
-        process_data [bool]: Whether to force data processing
-        document_results [bool]: Whether to force documenting
-        no_dirs [bool]: Whether to create files or not
-        verbose [int]: Level of verbosity
+        logging_level  : Optional[Union[int, str]]
```
Comment (tstreule) [inline Amplo/Pipeline.py:150]: Write instead: `int or str, optional`

File: Amplo/Utils/logging.py [hunk 6]
```diff
@@ -51,29 +51,54 @@ def _check_logging_level(level):
...
+
+        # Update last
+        self.last = record.relativeCreated
+
+        return True
+
+
 # Get custom logger
-logger = logging.getLogger('AutoML')
+logger = logging.getLogger('AmploML')
```
Comment (tstreule) [inline Amplo/Utils/logging.py:73]: Nice 😏😮‍💨

File: Amplo/AutoML/DataSampler.py (hunk 1 above)
Comment (nielsuit227) [reply Amplo/AutoML/DataSampler.py:180]: See cf9ddb3

File: Amplo/AutoML/IntervalAnalyser.py (hunk 2 above)
Comment (nielsuit227) [reply Amplo/AutoML/IntervalAnalyser.py:34]: [db41e4b](https://github.com/Amplo-GmbH/AutoML/commit/db41e4b3fb38dc920aac88db12ff9c240f2dd274)

File: Amplo/AutoML/IntervalAnalyser.py (hunk 3 above)
Comment (nielsuit227) [reply Amplo/AutoML/IntervalAnalyser.py:111]: 067a83c6c3a38ac82433fffbe6d690ebd77b1a7b

File: Amplo/Pipeline.py (hunk 4 above)
Comment (nielsuit227) [reply Amplo/Pipeline.py:105]: bb5a001

File: Amplo/AutoML/IntervalAnalyser.py (hunk 2 above)
Comment (tstreule) [reply Amplo/AutoML/IntervalAnalyser.py:34]: Wrong commit link

File: Amplo/Pipeline.py (hunk 5 above)
Comment (nielsuit227) [reply Amplo/Pipeline.py:150]: bb5a001

**Pull request merged**